docs/reference/buildx_build.md

```diff
@@ -15,7 +15,7 @@ Start a build
 
 | Name                                                                                                                                                   | Type          | Default   | Description                                                                                         |
 |:-------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------|:----------|:----------------------------------------------------------------------------------------------------|
-| [`--add-host`](https://docs.docker.com/engine/reference/commandline/build/#add-host)                                                                   | `stringSlice` |           | Add a custom host-to-IP mapping (format: `host:ip`)                                                 |
+| [`--add-host`](https://docs.docker.com/engine/reference/commandline/build/#add-host)                                                                   | `stringSlice` |           | Add a custom host-to-IP mapping (format: `host=ip`)                                                 |
```

Comment:
I'm not sure if we should just switch the formatting in docs as it takes time for people to upgrade to the latest versions. Doesn't look like the old formatting is deprecated as well.

Reply:
Yes, I see what you mean - the online docs aren't very version-specific.

This part of the ".md" is generated from the flags.StringSliceVar() text.  We could put both in there with an "or", but that may not really help to make it obvious that the `:` is for backward compatibility - and we probably don't want an essay in there. Earlier comments have been about preferring ("soft-deprecating") the colon separator.

On the other hand, the "--add-host" in the first column here is a link to the cli/build docs, here it is in my branch ...

https://github.com/robmry/buildx/blob/align_add-host_with_cli/docs/reference/buildx_build.md

And the modified version of the page it'll link to is here ...

https://github.com/robmry/cli/blob/4648-clearer_ipv6_in_add-host/docs/reference/commandline/build.md#add-host

It says ...

> You can wrap an IPv6 address in square brackets. `=` and `:` are both valid separators. Both formats in the following example are valid:

... perhaps we should make it clear there that the `:` is for backward compatibility?

@dvdksn - any suggestions?